### PR TITLE
CORD-19 06/12 update

### PR DESCRIPTION
**Pull request by edwinzhng**
Title

### CODE REVIEW

File: client/src/components/pages/HomePage/HomeText.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const HomeText = () => {
         <Link href="https://allenai.org/" target="_blank" rel="noopener noreferrer">
           Allen Institute for AI
         </Link>{' '}
-        (data release of May 26, 2020), which currently contains over 47,000 scholarly articles,
+        (data release of June 12, 2020), which currently contains over 47,000 scholarly articles,
```
Comment (rodrigonogueira4) [inline client/src/components/pages/HomePage/HomeText.tsx:22]: it has more than 47k, right?

Comment (edwinzhng) [reply client/src/components/pages/HomePage/HomeText.tsx:22]: website says 50k, will update

Comment (edwinzhng) [reply client/src/components/pages/HomePage/HomeText.tsx:22]: in the description here: https://www.semanticscholar.org/cord19/download

Comment (rodrigonogueira4) [reply client/src/components/pages/HomePage/HomeText.tsx]: I think it is ~120k (with ~60k full texts), the website must be outdated...

Comment (edwinzhng) [reply client/src/components/pages/HomePage/HomeText.tsx]: How about I just remove this sentence, I don't think it really adds much

Comment (lintool) [reply client/src/components/pages/HomePage/HomeText.tsx]: Let's go by: https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/2020-06-12/changelog

```
---SUMMARY---
total metadata rows: 140780
CORD UIDs (new: 280, removed: 1)

Full text:
  PDF - 72027 json (new: 258, removed: 1943)
  PMC - 52614 json (new: 270, removed: 1)
```

I'd just say 140k articles. I think it's still important to give the user a sense of the size of the corpus.

Comment (edwinzhng) [reply client/src/components/pages/HomePage/HomeText.tsx]: updated to 140k

**Pull request merged**